packages/third-parties/package.json

```diff
@@ -6,18 +6,22 @@
     "url": "vercel/next.js",
     "directory": "packages/third-parties"
   },
-  "main": "dist/index.js",
+  "exports": {
+    ".": "./dist/index.js",
+    "./google": "./dist/google/index.js"
+  },
   "files": [
     "dist"
   ],
   "license": "MIT",
   "scripts": {
-    "manual-build": "rm -rf dist && tsc -d -p tsconfig.json && node scripts/update-third-parties",
+    "build": "rm -rf src/**/index.tsx && rm -rf dist && tsc -d -p tsconfig.json && node scripts/update-third-parties",
```

Comment:
Why is the build script modifying the src folder? This will break publishes as it creates changes in the working tree